src/components/extractionQc/ExtractionQcForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div>
+    <traction-form class="text-left" @submit="onSubmit">
+      <traction-sub-section title="Which QC Results would you like to upload?" class="py-6">
+        <traction-select
+          v-model="usedBySelected"
+          :options="usedByOptions"
+          :state="Boolean(usedBySelected) ? true : null"
```

Comment:
state can be simplified to !!usedBySelected

Reply:
I can replace it with `!!(usedBySelected) ? true : null` if that's any nicer, but not `!!usedBySelected`, as we want to return null instead of false, otherwise the select appears red by default